models/dvmdostem/inst/config.js.template

```diff
@@ -0,0 +1,85 @@
+{
+  "general": {
+    "run_name": "A sample dvmdostem run. Modify this text to suit your needs."
+  },
+
+  "IO": {
+    "parameter_dir":      "parameters/",
+    "hist_climate_file":  "@INPUT_DATA_DIR@/historic-climate.nc",
+    "proj_climate_file":  "@INPUT_DATA_DIR@/projected-climate.nc",
+    "veg_class_file":     "@INPUT_DATA_DIR@/vegetation.nc",
+    "drainage_file":      "@INPUT_DATA_DIR@/drainage.nc",
+    "soil_texture_file":  "@INPUT_DATA_DIR@/soil-texture.nc",
+    "co2_file":           "@INPUT_DATA_DIR@/co2.nc",
```

Comment:
Of the things in this list, PEcAn should be able to provide climate, veg, soil texture, and co2 at the moment -- most everything else will need to have model defaults in the near-term. However, you probably want to template the file names themselves as well since there's no guarantee that the name of the file PEcAn is passing in will be the same as the one listed here. Also, does TEM strictly require that all these files be in the same directory? PEcAn will have them in different directories by default and thus it'd be simpler to just have these paths be where pecan has the files stored (which it will provide you under settings$run$inputs), rather than adding the extra step of creating a bunch of symlinks in INPUT_DATA_DIR